qiskit/circuit/library/data_preparation/pauli_feature_map.py

```diff
@@ -106,6 +106,7 @@ class PauliFeatureMap(NLocal):
     def __init__(self,
                  feature_dimension: Optional[int] = None,
                  reps: int = 2,
+                 alpha: float = 2.0,
                  entanglement: Union[str, List[List[int]], Callable[[int], List[int]]] = 'full',
```

Comment:
I'd move this one down so we have a more consistent ordering of parameters in all NLocal circuits (ie the first three args are num qubits, reps and entanglement)
```suggestion
                 entanglement: Union[str, List[List[int]], Callable[[int], List[int]]] = 'full',
                 alpha: float = 2.0,
```